<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentpage</class>
 <widget class="QWidget" name="studentpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>628</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1072</width>
    <height>628</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1072</width>
    <height>628</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(235, 244, 234);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <fontweight>Black</fontweight>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 900 16pt &quot;Arial Black&quot;;
background-color: rgb(0, 0, 0);
background-color: rgb(22, 102, 57);</string>
   </property>
   <property name="text">
    <string>学生管理界面</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>118</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <item>
    <property name="text">
     <string>按学生姓名查询</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按学号查询</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按所属院系查询</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按性别查询</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>查看全部</string>
    </property>
   </item>
  </widget>
  <widget class="QTableView" name="studentTable">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>521</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>351</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(22, 102, 57);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <widget class="QPushButton" name="toclassBtn">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(235, 244, 234);</string>
      </property>
      <property name="text">
       <string>管理课程</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="toteacherBtn">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(235, 244, 234);</string>
      </property>
      <property name="text">
       <string>管理教师</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="toanounceBtn">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(235, 244, 234);</string>
      </property>
      <property name="text">
       <string>发布公告</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="key_Edit">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>118</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>请输入查询信息：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchBtn">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>118</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
image: url(:/image/image/search.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="deleterow">
   <property name="geometry">
    <rect>
     <x>497</x>
     <y>120</y>
     <width>83</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string> 删除整行</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="searchbox">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>130</y>
     <width>341</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>新建</string>
    </attribute>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>42</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>电话</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>住址</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>166</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="phone_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="add_Edit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>66</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>学号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>性别</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>所属院系</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>12</y>
       <width>171</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="id_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="pass_Edit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="name_Edit"/>
      </item>
      <item>
       <widget class="QComboBox" name="sexBox_2">
        <item>
         <property name="text">
          <string>男</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>女</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dep_Edit"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="refreshBtn">
     <property name="geometry">
      <rect>
       <x>229</x>
       <y>240</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget">
    <attribute name="title">
     <string>修改</string>
    </attribute>
    <widget class="QWidget" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>71</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>学号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>性别</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>所属院系</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_7" native="true">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>151</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLineEdit" name="idEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
      <item>
       <widget class="QComboBox" name="sexBox">
        <item>
         <property name="text">
          <string>男</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>女</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="depEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_8" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>42</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>电话</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>住址</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_9" native="true">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLineEdit" name="phoneEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="addEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="confirmBtn">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>240</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>保存修改</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>删除</string>
    </attribute>
    <widget class="QLineEdit" name="deleteEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入学生id：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteBtn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>641</x>
     <y>97</y>
     <width>55</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>711</x>
     <y>97</y>
     <width>55</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>782</x>
     <y>97</y>
     <width>55</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scoreBtn">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>578</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QTableView" name="scoretable">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>464</y>
     <width>341</width>
     <height>148</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="topLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>85</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(17, 99, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="picLabel">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(17, 99, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>443</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>学生成绩：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitBtn">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>退出
登录</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
